packages/app/src/sections/dashboard/FuturesPositionsTable/FuturesPositionsTable.tsx

```diff
@@ -101,15 +106,19 @@ const FuturesPositionsTable: FC<FuturesPositionTableProps> = ({
 											<IconContainer>
 												<StyledCurrencyIcon currencyKey={cellProps.row.original.market.marketKey} />
 											</IconContainer>
-											<StyledText>
-												{cellProps.row.original.market.marketName}
-												<MarketBadge
-													currencyKey={cellProps.row.original.market.marketKey}
-													isFuturesMarketClosed={cellProps.row.original.market.isSuspended}
-													futuresClosureReason={cellProps.row.original.market.marketClosureReason}
-												/>
-											</StyledText>
-											<StyledValue>{cellProps.row.original.description}</StyledValue>
+											<CellContainer>
+												<StyledText>
+													{cellProps.row.original.market.marketName}
+													<MarketBadge
+														currencyKey={cellProps.row.original.market.marketKey}
+														isFuturesMarketClosed={cellProps.row.original.market.isSuspended}
+														futuresClosureReason={cellProps.row.original.market.marketClosureReason}
+													/>
+												</StyledText>
+												<StyledValue>
+													<Currency.Price price={cellProps.row.original.marketPrice} colored />
```

Comment:
I think we'll want to use `ColoredPrice` and pass `priceChange` prop so it can react as the price changes direction. Currently this will just always stay green.